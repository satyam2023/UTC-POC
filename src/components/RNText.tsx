import { StyleSheet, Text, View } from 'react-native'
import React from 'react';

interface IRnText{
  name:string;
}

const RNText = ({name}:IRnText) => {
  return (
    <View>
      <Text>Hi {name}</Text>
    </View>
  )
}

export default RNText

const styles = StyleSheet.create({})