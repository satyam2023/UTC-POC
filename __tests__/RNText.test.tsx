
import renderer from 'react-test-renderer';
import RNText from '../src/components/RNText';

describe("",()=>{
    test("",()=>{
    const component =renderer.create(<RNText name={'Satyam'}/>);
    expect(component.toJSON()).toMatchSnapshot();
    });
})