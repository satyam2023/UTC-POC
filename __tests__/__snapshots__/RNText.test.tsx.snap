// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`  1`] = `
<View>
  <Text>
    Hi 
    Satyam
  </Text>
</View>
`;
